### PR TITLE
added GenericService and GenericDao

### PR DESCRIPTION
**Pull request by MaksymMatuliak**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/controllers/IndexController.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class IndexController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/IndexController.java]: your line should be not longer than 100 symbols

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<h1>Hello World</h1>
+</body>
+</html>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/index.jsp]: add empty line to the EOF

**Review by dmytro-hryhoruk — COMMENTED** [review]
let's set up travis ci to check your PR every time https://www.youtube.com/watch?v=BHSsWTbuow4

File: src/main/java/mate/academy/internetshop/controllers/IndexController.java (hunk 1 above)
Comment (MaksymMatuliak) [reply src/main/java/mate/academy/internetshop/controllers/IndexController.java]: done

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 2 above)
Comment (MaksymMatuliak) [reply src/main/webapp/WEB-INF/views/index.jsp]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**